dd-trace/src/configuration/configuration.rs

```diff
@@ -436,26 +670,26 @@ pub struct Config {
     language: &'static str,
 
     // # Service tagging
-    service: ServiceName,
-    env: Option<String>,
-    version: Option<String>,
+    service: ConfigItem<ServiceName>,
+    env: ConfigItem<OptionalString>,
```

Comment:
I had to create all these types (`OptionalString`, `GlobalTags`, `TracePropagationStyleList`) to implement `Display`